pkg/util/mount/mount_windows.go

```diff
@@ -74,8 +74,23 @@ func (mounter *Mounter) Mount(source string, target string, fstype string, optio
 			return nil
 		}
 
-		// empty implementation for mounting azure file
-		return os.MkdirAll(target, 0755)
+		cmd := fmt.Sprintf(`$User = "AZURE\%s";$PWord = ConvertTo-SecureString -String "%s" -AsPlainText -Force;`+
```

Comment:
what if it is an nfs mount?

Reply:
currently only SMB(a nfs protocol) is supported for windows container in the new Windows 2016 RS3 image. It's only by New-SmbGlobalMapping cmdlet that windows container could use for volume mapping, "net use" command does not work for windows container.